config/docker/security/rotate-secrets.sh

```
@@ -0,0 +1,110 @@
+    openssl req -x509 -new -nodes \
+        -key "$SECRETS_DIR/ssl_key.pem" \
+        -sha256 -days 365 \
+        -out "$SECRETS_DIR/ssl_cert.pem" \
+        -subj "/C=US/ST=State/L=City/O=Organization/CN=localhost"
+    
+    # Update Vault if available
+    if [ -n "$VAULT_TOKEN" ] && [ -n "$VAULT_ADDR" ]; then
+        curl -s -X POST \
+            -H "X-Vault-Token: $VAULT_TOKEN" \
+            -H "Content-Type: application/json" \
+            -d @- "$VAULT_ADDR/v1/secret/ssl/cert" << EOF
```

Shellcheck (0.10.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)